### ISSUE TITLE
Disable custom mark as default

### ISSUE THREAD

**Issue by bogdancondorachi** [johannschopplich/kirby-writer-marks#9]
Hi @johannschopplich, I wonder if there's the posibility to not include all the custom marks as default on all the writer fields, and be able to add them depending on where you want with the `mark:` option in blueprints.

It would be much simplier to add all the custom marks only where I want then to have to use the `mark:` option on each writer field to remove the marks that I don't need.

Thanks for this awesome plugin :)

**Comment by johannschopplich** [comment]
Good point, that would indeed make sense. Since I'm focussed on other projects at the moment apart from custom writer fields, I probably won't have the resources to implement your request. PR is happily welcome!
